src/encryption.rs

```diff
@@ -116,6 +116,51 @@ impl From<matrix_sdk_crypto::types::EventEncryptionAlgorithm> for EncryptionAlgo
     }
 }
 
+#[wasm_bindgen()]
+#[derive(Debug, Clone, PartialEq, Eq)]
+/// Strategy to collect the devices that should receive room keys for the
```

Comment:
Please put the `#[…]` after the documentation. It's more Rust idiomatic to do so.

Reply:
done